lib/log-config.js

```diff
@@ -1,10 +1,14 @@
 var bunyan = require('bunyan');
+var PrettyStream = require('bunyan-prettystream');
```

Comment:
it's makes reading debug messages waaaaaaay easier..

we can only use it if node_env !== production